docs/dashboards/datagrid-component.md

```diff
@@ -0,0 +1,104 @@
+DataGrid Component
+===
+
+The DataGrid Component can be placed inside a data cell to allow the data visualization in the form of an editable data grid.
```

Comment:
`a data cell` we don't use this phrase. It's just `a cell` or if you would like to be specific it's `a dashboard's cell`.

Reply:
`allow user to visualize the data in the editable table` ?

Just a suggestion, because for me, the description is too complicated.